benchmark/benchmark.go

```diff
@@ -104,6 +108,36 @@ func (s *testServer) StreamingCall(stream testpb.BenchmarkService_StreamingCallS
 	}
 }
 
+func (s *testServer) UnconstrainedStreamingCall(stream testpb.BenchmarkService_UnconstrainedStreamingCallServer) error {
+	in := new(testpb.SimpleRequest)
+	go func() {
+		for {
+			if err := stream.(grpc.ServerStream).RecvMsg(in); err != nil {
+				return
+			}
+		}
+	}()
+
+	response := &testpb.SimpleResponse{
+		Payload: new(testpb.Payload),
+	}
+	setPayload(response.Payload, testpb.PayloadType_COMPRESSABLE, ResponseSize)
```

Comment:
The request messages have a `ResponseSize` field in them (and a `Type`).  We could use the first request's size/type, or we could delete the field entirely and configure the server explicitly rather than using the request.